docs/pages/world/account-delegation.mdx

```diff
@@ -21,9 +23,68 @@ This feature enables multiple use cases.
 
 The most common type of delegation is when a delegator address allows a specific delegatee address to act on its behalf.
 
+### Deploy delegation
+
+The owner of the root namespace (and therefore the `World`) needs to deploy the [`StandardDelegationModule`](https://github.com/latticexyz/mud/blob/main/packages/world-modules/src/modules/std-delegations/StandardDelegationsModule.sol).
+
+<CollapseCode>
+
+```solidity filename="DeployDelegation.s.sol" copy showLineNumbers {19-20}
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.21;
+
+import { Script } from "forge-std/Script.sol";
+import { console } from "forge-std/console.sol";
+
+import { IWorld } from "../src/codegen/world/IWorld.sol";
+
+import { StandardDelegationsModule } from "@latticexyz/world-modules/src/modules/std-delegations/StandardDelegationsModule.sol";
+
+contract DeployDelegation is Script {
+  function run() external {
+    uint256 deployerPrivateKey = vm.envUint("PRIVATE_KEY");
+    address worldAddress = vm.envAddress("WORLD_ADDRESS");
+
+    vm.startBroadcast(deployerPrivateKey);
+    IWorld world = IWorld(worldAddress);
+
+    StandardDelegationsModule standardDelegationsModule = new StandardDelegationsModule();
+    world.installRootModule(standardDelegationsModule, new bytes(0));
+
+    vm.stopBroadcast();
+  }
+}
+```
+
+</CollapseCode>
+
+<details>
+
+<summary>Explanation</summary>
+
+Other than boilerplate, this script does two things.
+
+```solidity
+StandardDelegationsModule standardDelegationsModule =
+  new StandardDelegationsModule();
+```
+
+Deploy a new `StandardDelegationsModule` contract.
+This contract, similar to a `System` contract, is stateless.
```

Comment:
I think this line is a bit confusing - we should teach folks to think about Modules as "onchain installation scripts" that install things into the World. This module installs two things - the `TimeboundDelegation` check and the `SystemboundDelegation`, both of which are systems that follow the `DelegationControl` interface.